core/misc/tableselect.js

```diff
@@ -20,6 +20,8 @@ Backdrop.tableSelect = function () {
     // Update table's select-all checkbox (and sticky header's if available).
     $(table).prev('table.sticky-header').addBack().find('th.select-all input[type="checkbox"]').each(function() {
       $(this).attr('title', state ? strings.selectNone : strings.selectAll);
+      $(this).toggleClass('select-none', state);
+      $(this).toggleClass('select-all', !state);
```

Comment:
Is this a separate bug-fix?

Reply:
Nope. Just adding a way to directly and explicitly select the "check all" checkbox in the js that I've added in installer.select_updates.js.

You see, the "select all" checkbox normally only has a `.form-checkbox` class, which also matches any other checkbox in the table. There is a `.select-all` class, but that is being added in the `<th>` that wraps the checkbox, instead of the checkbox itself.

Anyway, I've reverted this, and used `.select-all .form-checkbox:checked` as a selector in my js.